projects/aws/upgrader/upgrade.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/sh
```

Comment:
im not reviewing this file right now since I expect it to change significantly while yall dev this feature.  I think this may be something that should be a go binary instead of bash script, but please discuss and work out a plan on yalls side.